src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const regularCashBack = 20;
+  const bigCashBack = 50;
+  const daysForRegular = 3;
+  const daysForBig = 7;
+
+  if (days < daysForRegular) {
+    return days * pricePerDay;
+  }
+
+  if (days >= daysForRegular && days < daysForBig) {
+    return days * pricePerDay - regularCashBack;
+  }
+
+  return days * pricePerDay - bigCashBack;
```

Comment:
You calculate `days * pricePerDay` three times.
Why not to do it once and use variable?